doc/usecases/gamification/gain_blossoms.md

```diff
@@ -0,0 +1,50 @@
+# Use Case: Gain Blossoms
```

Comment:
Yvonne has some concerns if we will be able to draw so many different blossoms. She will do a first attempt and we will see if we need to adjust to a larger group of prices (like any harvesting goods, where we will have many icons anyway...)

Reply:
I agree with you that the amount of icons might be a little high. Maybe we can reduce the number of fully unique blossoms and instead do a few base models and adding slight variations. We can also use this to indicate affiltiation to a specific track (e.g. all expert track blossoms are blue) and connection between individual milestones / milestone rareness (e.g. a blossom for 'honoring a map on national flower day' could have more pedals than 'honor your first map' but be the same base flower)